modules/apps/commerce/commerce-test/src/testIntegration/java/com/liferay/commerce/test/CommerceCheckoutTest.java

```diff
@@ -486,6 +807,45 @@ public void testUserCheckout() throws Exception {
 	@Rule
 	public FrutillaRule frutillaRule = new FrutillaRule();
 
+	protected CommerceAddress addCommerceAddress(
+			CommerceOrder commerceOrder, int addressType)
+		throws Exception {
+
+		String a2 = RandomTestUtil.randomString(2);
+		String a3 = RandomTestUtil.randomString(3);
+
+		Country country = CountryLocalServiceUtil.fetchCountryByA2(
+			commerceOrder.getCompanyId(), a2);
+
+		if (country == null) {
+			country = CountryLocalServiceUtil.fetchCountryByA3(
+				commerceOrder.getCompanyId(), a3);
+		}
+
+		if (country == null) {
+			country = CountryLocalServiceUtil.addCountry(
+				a2, a3, true, true, RandomTestUtil.randomString(),
+				RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+				0D, true, true, false, _serviceContext);
+		}
+
+		return CommerceAddressLocalServiceUtil.addCommerceAddress(
+			CommerceAccount.class.getName(),
+			commerceOrder.getCommerceAccountId(),
+			RandomTestUtil.randomString(), //name
+			RandomTestUtil.randomString(), //description
+			RandomTestUtil.randomString(), //stree1
+			RandomTestUtil.randomString(), //street2
+			RandomTestUtil.randomString(), //street3
+			RandomTestUtil.randomString(), //city
+			RandomTestUtil.randomString(), //zip
+			0, //regionId
+			country.getCountryId(), //countryId
+			RandomTestUtil.randomString(), //phoneNumber
```

Comment:
remove comments when not necessary
```suggestion
			RandomTestUtil.randomString(),
			RandomTestUtil.randomString(),
			RandomTestUtil.randomString(),
			RandomTestUtil.randomString(),
			RandomTestUtil.randomString(),
			RandomTestUtil.randomString(),
			RandomTestUtil.randomString(),
			0,
			country.getCountryId(),
			RandomTestUtil.randomString(),
```